source/use-cases/home-resolver.rst

```diff
@@ -1,6 +1,200 @@
+
+.. code-block:: bash
+
+	server:
+			# location of the trust anchor file that enables DNSSEC
+			auto-trust-anchor-file: "/var/lib/unbound/root.key"
+			# the interface that is used to connect to the network, this means on this machine
+			interface: 0.0.0.0
+			# interface: ::0
+			# addresses from the IP range that are allowed to connect to the resolver
+			access-control: 10.0.0.0/8 allow
+			# access-control: 2001:DB8.. code-block:: bash/64 allow
+	remote-control:
+			control-enable: yes
+
+The access-control is currently configured to listen to any address on the machine, and only allow queries from the ``10.0.0.0/8`` `IP subnet <https://www.ripe.net/about-us/press-centre/understanding-ip-addressing>`_ range. Note that the IP addresses we chose (``10.0.0.1`` and ``10.0.0.2``) fall within the ``10.0.0.0/8`` range.
+
+To prepare our config we are going to modify the existing config in :file:`/etc/unbound/unbound.conf`.
+If you open the file we see that there is already an “include” in there. This include enables us to do `DNSSEC <https://en.wikipedia.org/wiki/Domain_Name_System_Security_Extensions>`_, which allows Unbound to verify the source of the answers that it receives, which we want to keep in. If you don't have the files that the include links to, they can be created using the :command:`unbound-anchor` command.
+
+Using the text editor again, we can then add the minimal config as shown above, making any changes to the access control where needed. Do note that we strongly recommend keeping the :command:`include` that is already in the file. We also add the :command:`remote-control` in the config to enable controlling Unbound using :command:`unbound-control` command which is useful if you want to modify the config later on. When you are happy with your config, we first need to stop the currently running Unbound server and restart it with our new configuration. You can stop the currently running instance with:
+
+.. code-block:: bash
+
+	pkill -f unbound
+
+And you can restart Unbound with:
+
+.. code-block:: bash
+
+	unbound -c /etc/unbound.conf
+
+From this point on, we can stop, start, and reload the instance with :command:`unbound-control` if you want to make changes to the configuration.
+
+Testing the resolver from a remote machine
+------------------------------------------
+
+So now we have a DNS resolver which should be reachable from within the network. To verify this we need to find the IP address of the resolver machine which can be found on the machine itself. For this tutorial we will use the address ``10.0.0.2`` (not ``127.0.0.1`` as we saw earlier) as an example. Armed with the IP address we can send a query to our DNS resolver from another machine which is within our home network. To do this we use the same dig command, only we change the IP address where the query is asked.
```

Comment:
Should there be a mention about firewalls somewhere, e.g. if the O/S by default blocks external connections to port 53 it will not be possible to reach the resolver from another machine?